bogota-abril-2019/css-magazine-cover/laura_bruges-task2-magazine/styles/magazine.css

```diff
@@ -0,0 +1,167 @@
+#margin {
+    position: absolute;
+    height: 95vh;

```

Comment:
Curious: Why these units?

Reply:
I was trying to make it more responsive (has an odd behavior tho)